experiments/impttest.py

```diff
@@ -5,70 +5,49 @@
+
+def phase_encode(beam_iter, sequence_num, num_pulses):
```

Comment:
Yes but the function signature is required to have them. It's how we set up the pulse phase encoding to work - the experiment can define a function that takes the above parameters and returns phases for each pulse. The parameter list was set to (hopefully) give the function enough inputs that it could be very flexible, but this specific function in this experiment doesn't need the flexibility.